torch/_inductor/codecache.py

```diff
@@ -2095,7 +2096,7 @@ def process_pool() -> ProcessPoolExecutor:
         # doesn't run, and we need to register our own handler.
         # exitpriority has to be high, because another one of the finalizers will
         # kill the worker thread that sends the shutdown message to the workers...
-        multiprocessing.util.Finalize(None, pool.shutdown, exitpriority=sys.maxsize)
+        multiprocessing.util.Finalize(None, pool.shutdown, exitpriority=sys.maxsize)  # type: ignore[attr-defined]
```

Comment:
Can we move these type annotations to another PR in the stack?

Reply:
Without them, I had linter errors. Not really sure why, because I did not change these lines originally. I will try removing them again and see ..

Reply:
Reverted the changes to that file to simplify review